crates/ty_python_semantic/resources/mdtest/narrow/match.md

```diff
@@ -131,13 +160,14 @@ reveal_type(x)  # revealed: object
 
 match x:
     case "foo" | 42 | None:
-        reveal_type(x)  # revealed: Literal["foo", 42] | None
+        reveal_type(x)  # revealed: object
```

Comment:
Similar here. Can we maybe update the type of `x` (to a large union) to restore the spirit of this test.